ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -141,6 +159,10 @@ public StoreConfig(VerifiableProperties verifiableProperties) {
     storeEnableCompaction = verifiableProperties.getBoolean("store.enable.compaction", false);
     storeCompactionCheckFrequencyInHours =
         verifiableProperties.getIntInRange("store.compaction.check.frequency.in.hours", 7 * 24, 1, Integer.MAX_VALUE);
+    storeStatsBucketCount = verifiableProperties.getInt("store.stats.bucket.count", 24);
+    storeStatsBucketSpanInMs = verifiableProperties.getLong("store.stats.bucket.span.in.ms", 3600000);
+    storeStatsQueueProcessorPeriodInMs =
+        verifiableProperties.getLong("store.stats.queue.processor.period.in.ms", 120000);
```

Comment:
always specify defaults in human readable form. for eg: you could represent 1 day in ms as 1000 * 60 * 60 * 24